transmart-core-db-tests/test/integration/org/transmartproject/db/ontology/OntologyTermTagsResourceServiceTests.groovy

```diff
@@ -118,13 +118,25 @@ class OntologyTermTagsResourceServiceTests {
                     hasProperty('key', equalTo(key)),
```

Comment:
I am confused: `key` variable has `String` type. But map signature suggests that the map contains an object of the `OntologyTerm` class.